mne/decoding/tests/test_search_light.py

```diff
@@ -254,6 +254,33 @@ def test_generalization_light():
     assert_array_equal(y_preds[0], y_preds[1])
 
 
+@requires_sklearn
+def test_verbose_arg(capsys):
+    """Test controlling output with the ``verbose`` argument."""
+    from sklearn.svm import SVC
+
+    X, y = make_data()
+    clf = SVC()
+
+    # shows progress bar and prints other messages to the console
+    with use_log_level(True):
+        for estimator_object in [SlidingEstimator, GeneralizingEstimator]:
+            for n_jobs, verbose in zip([1, 2, 1, 2],
+                                       [False, False, True, 'info']):
```

Comment:
Unless `make_data` has bad overhead (in which case you could make it a module-level fixture!) I would `@pytest.mark.parametrize('n_jobs, verbose', [(1, False), (2, False), (1, True), (2, 'info')])` this. It saves an indentation level and allows all versions to run even if one has an error, which makes debugging failures later (e.g., "okay these two fail but the other two succeed...").